src/Adapter/DoctrineAdapter.php

```diff
@@ -707,7 +707,9 @@ public function setAuthorizationCode(
         $redirect_uri,
         $expires,
         $scope = null,
-        $id_token = null
+        $id_token = null,
```

Comment:
Note that this is just an interface update to make things compatible. A proper fix would be to include the two new fields in the list of fields sent to the database, but that requires a lot more code tweaks. Without upstream changes (which most likely will never arrive), I think it's best to at least keep it compatible with the new interface from the `bshaffer/oauth2-server-php` package to not block other updates.